paymentsheet/src/test/java/com/stripe/android/paymentsheet/VerticalModePaymentSheetActivityTest.kt

```diff
@@ -196,17 +274,50 @@ internal class VerticalModePaymentSheetActivityTest {
         }
     }
 
-    private fun setupV1PaymentMethodsResponse(isEmpty: Boolean) {
+    private fun setupV1PaymentMethodsResponse(count: Int) {
```

Comment:
could `count` be named something more descriptive here? I wasn't sure if this meant the number of responses, but it seems to mean the number of PMs

Reply:
Or actually -- what do you think of passing in the payment method IDs here instead of the count? Since the IDs are defined in this function and then used in the tests, I think someone could end up making a change to this function that breaks tests. I think it'd be easier to reason about if, in each test, you pass the IDs you expect to be returned and then those are used. Would probably simplify this code too

Reply:
Mind if I do this in a follow up or two? I've got another change ready for review that builds on this 🙈 

I introduce a new abstraction there. But I'd be happy to follow that change up to make it easy to reason about this!

Reply:
sgtm!